packages/next/next-server/server/next-server.ts

```diff
@@ -1029,6 +1029,7 @@ export default class Server {
     // we lazy load the staticPaths to prevent the user
     // from waiting on them for the page to load in dev mode
     let staticPaths: string[] | undefined
+    let hasStaticFallback = false
```

Comment:
We need to get this from the prerender-manifest in production right?

Reply:
Good call!